packages/headless/src/api/commerce/common/request.ts

```diff
@@ -56,7 +56,7 @@ const prepareRequestParams = (req: CommerceAPIRequest) => {
 };
 
 export const baseRequest = (
-  req: BaseParam & TrackingIdParam,
```

Comment:
It's already in the `CommerceAPIRequest` type; doesn't need to be there as well.